parachain-template/runtime/src/xcm_config.rs

```diff
@@ -203,8 +203,8 @@ impl pallet_xcm::Config for Runtime {
 	// ^ Disable dispatchable execute on the XCM pallet.
 	// Needs to be `Everything` for local testing.
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Nothing;
+	type XcmTeleportFilter = Nothing;
+	type XcmReserveTransferFilter = Everything;
```

Comment:
Would be better with some comments on what these settings are doing?

Reply:
Dumb q: why should you not do this? And if you do, what would the effect be? 

Assuming it's something that should (almost) never happen, is there any check possible to ensure that there are no collisions in these settings, something like `Contains` impl for these so they explicitly exclude each other could help you from screwing this up.

Reply:
Because an attacker can drain the `CheckedAccount` on the reserve location by doing a reserve transfer to the parachain and then teleporting the asset back. This will then prevent others from teleporting back to the reserve location.